index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Web App – Simple Interest Calculator</title>
+</head>
+
+<body>
+    <div class="maindiv">
+        <h1>Simple Interest Calculator</h1>
+        <div class="form">
+            <div class="form-input">
+                <label for="principal">Amount</label> 
+                <input type="number" id="principal">
+            </div>
+            <div class="form-input">
+                <label for="rate">Rate</label> 
+                 <input onchange="updateRate()" type="range" id="rate" step="0.25" min="1" max="20" value="10.25"> <span
+                id="rate_val"> 10.25 %
+            </span>
+            </div>
+            <div class="form-input">
+                <label for="years">No. of Years</label> 
+                 <select id="years">
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4">4</option>
+                    <option value="5">5</option>
+                    <option value="6">6</option>
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+    
```

Comment:
- [ ]  Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.
checkout [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)